src/gestures-hooks.tsx

```diff
@@ -134,6 +134,10 @@ class GestureActions {
     if (!this.gestureState.data.length) {
       this.setGestureState({ data: [this.generateNewGesture(true)] });
     }
+    // If icon is missing from stored data, generate default icons.
+    if (this.gestureState.data.some((g) => !g.icon)) {
+      this.validateAndSetGestures(this.gestureState.data);
+    }
```

Comment:
The alternative option is to add `icon` to the `isValidStoredGestureData` function which would result in wiping the data and starting from scratch. This approach saves the existing data and pops the default icons in.